src/components/buttons/PrimaryBtn.tsx

```diff
@@ -0,0 +1,28 @@
+import { Add } from '@mui/icons-material'
+import { Button, Typography } from '@mui/material'
+import { ReactNode } from 'react'
+
+export const PrimaryBtn = ({
+  startIcon,
+  buttonText,
+  handleClick,
+}: {
+  startIcon?: ReactNode | undefined
```

Comment:
@aatbip the prop is already optional, do we need extra `undefined` type?

Reply:
removed in e31c33952a7e4fd9628d27e3593fb397273996a2 commit